agent/internal/client/mock.go

```diff
@@ -528,5 +534,8 @@ func (c *Mock) GetPullRequestInfo(ctx context.Context, taskData TaskData, prNum
 }
 
 func (c *Mock) CreateInstallationToken(ctx context.Context, td TaskData, owner, repo string) (string, error) {
-	return "token", nil
+	if c.CreateInstallationTokenFail {
+		return "", nil
```

Comment:
Seems a little odd to return nil error in the case of a "failure". Seems like it could be covered more nicely by the suggestion below.

Reply:
updated